src/main/java/Mikey.java

```diff
@@ -0,0 +1,295 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.StringJoiner;
+import java.util.Arrays;
+
+public class Mikey {
+
+    public static ArrayList<Task> tasks = new ArrayList<Task>();
+
+    public static Task newTodo(String taskName, int completion) {
+        Task newTask = null;
+        newTask = new Todo(taskName);
+        if(completion == 1) {
+            newTask.isDone = true;
+        } else {
+            newTask.isDone = false;
+        }
+        tasks.add(newTask);
+        return newTask;
+    }
+
+    public static Task newDeadline(String taskName, String dateDue, int completion) {
+        Task newTask = null;
+        newTask = new Deadline(taskName, dateDue);
+        if(completion == 1) {
+            newTask.isDone = true;
+        } else {
+            newTask.isDone = false;
+        }
+        tasks.add(newTask);
+        return newTask;
+    }
+
+    public static Task newEvent(String taskName, String timeOfEvent, int completion) {
+        Task newTask = null;
+        newTask = new Event(taskName, timeOfEvent);
+        if(completion == 1) {
+            newTask.isDone = true;
+        } else {
+            newTask.isDone = false;
+        }
+        tasks.add(newTask);
+        return newTask;
+    }
+    public static void printTask(int taskNumber) {
+        System.out.println("[" + tasks.get(taskNumber).getTaskType() + "]" + "["
+                + tasks.get(taskNumber).getStatusIcon() + "] " +  tasks.get(taskNumber).getName() + " "
+                    + tasks.get(taskNumber).getDate());
+        System.out.println();
+    }
+
+    public static void addTaskMessage() {
+        System.out.println("Now yous got " + tasks.size() + " thing(s) ta do in ya list");
+        System.out.println("Anythin else ya need me ta help ya with?");
+        System.out.println();
+    }
+
+    public static void deleteTask(int taskNumber) {
+        System.out.println("I've gotcha mate, removed [" + tasks.get(taskNumber).getTaskType() + "]" + "["
+            + tasks.get(taskNumber).getStatusIcon() + "]" + tasks.get(taskNumber).getName() + " "
+                + tasks.get(taskNumber).getDate());
+        tasks.remove(taskNumber);
```

Comment:
Since some of these lines exceed the 120-char limit, they've been wrapped well but maybe check if your indentation is correct. Indentation for wrapped lines should be 8 spaces (i.e. twice the normal indentation of 4 spaces) more than the parent line. [See here](https://se-education.org/guides/conventions/java/basic.html#layout)